config-server/src/main/resources/configurations/order-server.yml

```
@@ -0,0 +1,29 @@
+      value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
+      properties:
+        spring.json.type.mapping: orderConfirmationRequest:com.anup.ecommerce.dto.request.OrderConfirmationRequest
+
+application:
+  config:
+    customer-server:
+      url: http://localhost:8222/api/v1/customers
+    product-server:
+      url: http://localhost:8222/api/v1/products
+    payment-server:
+      url: http://localhost:8222/api/v1/payments
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)